salt/modules/win_file.py

```diff
@@ -788,6 +788,31 @@ def chgrp(path, group):
     return None
 
 
+def findstr(path, pattern):
```

Comment:
I think this should be renamed to `grep` since `win_file` shadows the functionality in the `file` execution module. It should also take additional opts as `grep` does.